src/clusterfuzz/_internal/platforms/android/flash.py

```diff
@@ -70,15 +70,18 @@ def download_latest_build(build_info, image_regexes, image_directory):
   # Clean up the images directory first.
   shell.remove_directory(image_directory, recreate=True)
   for image_regex in image_regexes:
-    image_file_path = fetch_artifact.get(build_id, target, image_regex,
+    image_file_paths = fetch_artifact.get(build_id, target, image_regex,
                                          image_directory)
-    if not image_file_path:
+
+    if not image_file_paths:
       logs.log_error('Failed to download artifact %s for '
                      'branch %s and target %s.' %
-                     (image_file_path, build_info['branch'], target))
+                     (image_file_paths, build_info['branch'], target))
       return
-    if image_file_path.endswith('.zip') or image_file_path.endswith('.tar.gz'):
-      archive.unpack(image_file_path, image_directory)
+
+    for file_path in image_file_paths:
+      if file_path.endswith('.zip') or file_path.endswith('.tar.gz'):
+        archive.unpack(file_path, image_directory)
```

Comment:
Would these all be part of the same logical "build" ? i.e. is it ever possible these unpacks will clash/overwrite each other in a weird way?

Reply:
Yes these would all be part of the same logical build (i.e. Coverage enabled Cuttlefish build for x86 or HWASAN cuttlefish build for ARM). I would be surprised if there was clashing of files like boot.img but it seems possible